Standards/scs-0402-v1-status-page-openapi-spec-decision.md

```diff
@@ -137,6 +135,12 @@ A severity with the value of 100 MUST always be supplied. This is the highest se
 
 Components list their impacts, which they are affected by, as read only. Only an incident creates an impact on a component. Components MUST only list their currently active impacts.
 
+An optional `now` parameter can be supplied, to set a reference time to show all incidents, active at that time, even when they are inactive currently.
```

Comment:
Maybe change the `now` bool parameter to be a timestamp named `at`? This will enable the same use case (and more), while keeping server responses stable, unaffected by system time changes.

Reply:
It is my understanding that this is already the intended behaviour. The name `at` is, however, a marked improvement in my eyes over `now`.

Reply:
changed